examples/app/Commands/CreateTableCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Commands;
+
+use App\AppService;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use YdbPlatform\Ydb\Session;
+
+class CreateTableCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'create';
+
+    /**
+     * @var AppService
+     */
+    protected $appService;
+
+    public function __construct()
+    {
+        $this->appService = new AppService;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Create a table.');
+
+        $this->addArgument('table', InputArgument::REQUIRED, 'The table name.');
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     * @throws \Exception
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $table_name = $input->getArgument('table') ?: '';
+
+        $columns = $this->getColumns();
+
+        $ydb = $this->appService->initYdb();
+
+        $ydb->table()->retrySession(function (Session $session){
+
+            $result = $session->createTable($table_name, $columns, 'id');
+
+            $output->writeln('Table ' . $table_name . ' has been created.');
+
+            $session->transaction(function($session) use ($table_name, $columns) {
+                $session->query('upsert into `' . $table_name . '` (`' . implode('`, `', array_keys($columns)) . '`) values (' . implode('), (', $this->getData()) . ');');
```

Comment:
insert data in text of query id bad example.

I don't see is it your changes or original code.